src/theories/Schema.jl

```diff
@@ -186,3 +189,18 @@ function attrs_by_codom(AD::Type{T}) where
 end
 
 SchemaType(pres::Presentation{Schema}) = (CatDescType(pres),AttrDescType(pres))
+
+"""
```

Comment:
In order to get a code block in the autogenerated docs you can add

```julia
    Presentation(CD::Type{T}, AD::Type{S}) where {T <: CatDesc, S <: AttrDesc}
```

to the first line of the docstring with 4 leading spaces (which is a code block in markdown)